services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int,) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall
+
+    )
```

Comment:
The function `create_movie_session` is defined to return `None`, but it currently returns the created `MovieSession` object. If the intention is to return the created object, update the function signature to reflect this change. Otherwise, remove the `return` statement to ensure the function returns `None` as specified.